__tests__/app.test.js

```diff
@@ -146,6 +146,14 @@ describe("/api/articles", () => {
             expect(body.articles).toBeSortedBy("created_at", { descending: true})
         })
     });
+    test('GET: 200 - articles can be ordered ascending', () => {
+        return request(app)
+          .get('/api/articles?order=asc')
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.articles).toBeSortedBy('created_at');
+          });
+    });
```

Comment:
A couple of untested behaviours here:

We could have an invalid order value (eg down)
Order or sort_by could be misspelled